gc/base/ObjectScanner.hpp

```diff
@@ -145,7 +151,8 @@ class GC_ObjectScanner : public MM_BaseVirtual
 	putNextSlotMapBit(fomrobject_t *nextSlotAddress)
```

Comment:
Nobody uses this method, please feel free to remove it. @rwy0717 Do you agree?

Reply:
Makes sense to me, this helper doesn't seem all that helpful :)

Reply:
Done.